stop_line:
  stop_margin: 0.0
